draft-ietf-quic-multipath.md

```diff
@@ -1376,16 +1376,20 @@ e.g. by pacing them out or limiting the number of path initiation attempts
 over a certain time period.
 
 
-## Transport Layer Security
+## Use of Transport Layer Security and the AEAD Encryption Nonce
 
 The multipath extension as specified in this document is only enabled after a
 successful handshake when both endpoints indicate support for this extension.
 Respectively, all new frames defined in this extension are only used in 1-RTT packets.
 As the handshake is not changed by this extension, the transport security mechanisms
 as specified in {{QUIC-TLS}}, such as encryption key exchange and peer authentication,
 remain unchanged as well and the respective security considerations in {{QUIC-TLS}} applied unaltered.
+Note that sepcficially also the limits as discussed on Appendix B of {{QUIC-TLS}}
+are unconfined valid and apply to all packets sent on any path. Further note,
+that with the use of this extension multiple nonces can be in use simulanteously,
+which need to be considered when tracking these limits and for key updates.
```

Comment:
Suggest to replace the text from "Notes" to "for keys updates." by:
Note that the limits as discussed on Appendix B of {{QUIC-TLS}} apply to the total number of packets sent on all paths. 

The reason is that we do not have a mathematical analysis of the effect of multiple parallel nonces on multiple paths. In the absence of that analysis, we have to take the most conservative position.

Reply:
Do you mean to only have the first sentence and remove the second sentence? I thought it important to note that multiple nonces are used at the same time and that's a difference to RFC9000.

Reply:
I was thinking of also removing the second sentence. As I said, I am not aware of a mathematical analysis showing that with multiple nonces the limits could be reconsidered.

Reply:
By the way, "any path" in the first sentence is a bit ambiguous. It by be safer to say "all packets sent on all paths." Or, "the total number of packets sent on all paths."

Reply:
The intention of this second sentence was not to say that limit might be different but that you need to be aware that you have to implementation your tracking such that the use of multiple nonces is considered because today maybe implementations assume only one active nonce.

Reply:
Oh, yes. I see. But maybe reorganize that so the issue of math limit and nonce implementation are not in the same paragraph?

Reply:
Maybe rewrite this as:

~~~
Note that with the use of this extension, multiple nonces can be in use simultaneously for the same AEAD key.

Further note that the limits as discussed on Appendix B of {{QUIC-TLS}}
apply to the total number of packets sent on any path.
~~~